tests/core/test_error_handling_agent.py

```diff
@@ -0,0 +1,189 @@
+# Standard Library
+import logging
+import os
+from pathlib import Path
+
+# Third Party
+import pytest
+
+# First Party
+from smdebug.core.error_handling_agent import BASE_ERROR_MESSAGE
+from smdebug.core.logger import DuplicateLogFilter, get_logger
+from smdebug.core.utils import error_handling_agent
+
+
+@pytest.fixture
+def stack_trace_filepath(out_dir):
```

Comment:
Try to avoid the duplication of these fixtures.

Reply:
Done